core/src/main/kotlin/org/evomaster/core/problem/graphql/IntrospectiveQuery.kt

```diff
@@ -61,35 +63,38 @@ class IntrospectiveQuery {
                         "query": "fragment FullType on __Type {  kind  name  fields(includeDeprecated: true) {    name    args {      ...InputValue    }   type {      ...TypeRef    }    isDeprecated    deprecationReason  }  inputFields {    ...InputValue  }  interfaces {    ...TypeRef  }  enumValues(includeDeprecated: true) {    name    isDeprecated    deprecationReason  }  possibleTypes {    ...TypeRef  }} fragment InputValue on __InputValue {  name type {   ...TypeRef  }  defaultValue}fragment TypeRef on __Type {  kind  name  ofType {    kind    name   ofType {      kind      name      ofType {        kind       name       ofType {          kind          name         ofType {            kind            name            ofType {             kind             name              ofType {                kind                name              }            }         }        }      }    }  }}query IntrospectionQuery {  __schema {    queryType {     name   }    mutationType {      name    }    types {      ...FullType   }    directives {      name      locations      args {        ...InputValue     }    } }}","variables":null,"operationName":"IntrospectionQuery"
                     }
                 """.trimIndent(), MediaType.APPLICATION_JSON_TYPE)
-
-        //TODO check if TCP problems
-        val response = SearchTimeController.measureTimeMillis({ ms, res ->
+        val response: Response?
```

Comment:
thx. in contrast to Java, in Kotlin most statements are expressions. so can write something directly like `val response = try {...`

Reply:
also why nullable with `?` ? 
if want to check case for null, could make it non-null, with explicit throw exception after `try` with the `?:` Elvis operator